lookout/style/format/analyzer.py

```diff
@@ -256,7 +256,7 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_service: D
             else:
                 _log.info("training on %d %s files", len(files), language)
             train_files, test_files = FormatAnalyzer.get_train_test_split(
-                files, lang_config["test_dataset_ratio"], random_state=random_state)
+                files.values(), lang_config["test_dataset_ratio"], random_state=random_state)
```

Comment:
`files` come here directly from `files_by_language` which does not sort them. `get_train_test_split` shuffles without prior sorting, too. We should either make `files_by_language` deterministic or sort in `get_train_test_split` before shuffling. I like the first option more.